tools/handle_ocp_versions.py

```diff
@@ -31,7 +31,7 @@ def verify_images(release_images: List[str]):
 
 def verify_release_version(ocp_version: str, release_image: str, release_version: str):
     oc_version = get_oc_version(release_image)
-    assert oc_version == release_version, f"{release_image} full version is {oc_version} not {release_version}"
+    assert oc_version == release_version or oc_version == release_version+"-multi", f"{release_image} full version is {oc_version} not {release_version}"
```

Comment:
pep8 dictates:
```suggestion
    assert oc_version == release_version or oc_version == release_version + "-multi", f"{release_image} full version is {oc_version} not {release_version}"
```
also let's split this line to make it more readable

Reply:
Sure, just fixed